content/docs/user-guide/remote-content/use-a-remote-repository.md

```diff
@@ -0,0 +1,163 @@
+---
+linkTitle: Using O3DE Remote Repositories 
+title: Using O3DE Remote Repositories
+description: O3DE remote repositories provide a way to share Gems, Projects and Templates in Open 3D Engine (O3DE).
+weight: 100
+toc: true
+---
+
+You can use the [Project Manager](#using-project-manager-to-access-o3de-remote-repositories#using-project-manager-to-access-o3de-remote-repositories) and [O3DE CLI](#using-the-o3de-command-line-tool-to-access-o3de-remote-repositories) to display,  download and register **Gems**, **Projects**, and **Templates** from **Remote Repositories**.
+
+## Using Project Manager to access O3DE remote repositories
+### Add a remote repository
+
+You can add existing O3DE remote repositories from the **Remote Sources** page in Project Manager which you can access from the **Engine** tab, or from the **Gem Catalog** right panel menu.
+1. Press the **Add Repository** button. In the dialogue box that opens you can supply the URL or local path for a Gem repository. 
```

Comment:
Should use `en-us` terms (https://languagetool.org/insights/post/spelling-dialog-or-dialogue/)
 ```suggestion
1. Press the **Add Repository** button. In the dialog box that opens you can supply the URL or local path for a Gem repository. 
```